pkg/configuration/base/validate.go

```diff
@@ -486,6 +503,63 @@ func (r *JenkinsBaseConfigurationReconciler) verifyBasePlugins(requiredBasePlugi
 	return messages
 }
 
+// Render executes a parsed template (go-template) with configuration from data.
+func Render(template *template.Template, data interface{}) (string, error) {
```

Comment:
Should this be a global function if this is the only place where it is used ? 
Can we rename this to executeTextTemplate if that is not the case ?

Reply:
executeTextTemplate tells us that we are executing a text template, the name "Render" is a bit ambiguous.